### PR TITLE
Change string when link is copied

### PR DESCRIPTION
**Pull request by nicobubulle**

### CODE REVIEW

File: src/locales/fr/ui.json [hunk 1]
```diff
@@ -492,5 +492,12 @@
     "Yes": "Oui",
     "No": "Non",
     "Don't show this again": "Ne plus afficher le message",
-    "Mars is Terraformed!": "Mars est terraformé !"
+    "Mars is Terraformed!": "Mars est terraformé !",
+    "Copy the text below and then paste it in": "Copiez le texte ci-dessous et collez-le soit",
+    "a": "dans une",
```
Comment (kberg) [inline src/locales/fr/ui.json:497]: I worry a little that translating the word "a" might make for some real translation issues.

Comment (nicobubulle) [reply src/locales/fr/ui.json:497]: I can include the "a" inside the <a> with "Github issue" ? What do you think ?

Comment (kberg) [reply src/locales/fr/ui.json:497]: Yeah. Let's do that.

**Comment by kberg** [comment]
@nicobubulle I think I'm waiting on you.

**Comment by nicobubulle** [comment]
Sorry, I was busy. Done !

**Comment by kberg** [comment]
My turn to apologize. This looks great. Thank you!

**Review by kberg — APPROVED** [review]

**Pull request merged**